x-pack/filebeat/module/zeek/mqtt_subscribe/_meta/fields.yml

```diff
@@ -0,0 +1,30 @@
+- name: mqtt_subscribe
+  type: group
+  default_field: false
+  description: >
+    Fields exported by the Zeek mqtt_subscribe log.
+  fields:
+    - name: action
```

Comment:
should this also get copied to event.action?

Reply:
what do you mean?

Reply:
@leehinman can you explain?

Reply:
Sorry I missed your earlier question.

event.action is an ECS [field](https://www.elastic.co/guide/en/ecs/current/ecs-event.html) that describes the action captured by the event.  It seems like this field is a good fit for that.  Much like id.orig_h maps to the ECS field source.address.  

We could either remove the custom field(zeek.mqtt_subscribe.action) and just use event.action or we could copy the value to event.action keeping the custom field (zeek.mqtt_subscribe.action).